pages/_document.js

```diff
@@ -1,27 +1,43 @@
-import Document from "next/document";
+import Document, { Html, Head, Main, NextScript } from "next/document";
 import { ServerStyleSheet } from "styled-components";
+import { i18n } from "../Utils/translation/i18n";
 export default class MyDocument extends Document {
   static async getInitialProps(ctx) {
     const sheet = new ServerStyleSheet();
     const originalRenderPage = ctx.renderPage;
+    const lang = ctx.req.language + "-GB";
```

Comment:
I don't think we need to append `-GB`

Reply:
Oh! I thought its for screen reader to know the variant english. I will remove that. Thanks

Reply:
Well, maybe check that because it might be useful - I'm looking at RAD for reference. Either way the bigger issue is that we don't see `lang="cy"` when we select the Welsh as a language option (and that wouldn't need `-GB`). Or maybe that's why it doesn't appear on RAD. I don't know. 
Also the TP card could have been much clearer on the requirement for this so I would query that.

Reply:
on RAD the lang attribute changes from lang="en" to lang="cy" when user switches to Welsh language. We require the same functionality here. I'll add this comment to the ticket.

Reply:
Well, on RAD every request go to the server, and a fresh copy of the document is sent. While in react all request are resolved in the client unless there is a deliberate refresh. The problem here is the document frame which contains the HTML tag is sent once and for all from the nextjs server (while others can be change in the client), so, to change the attribute on the tag, it must be through a fresh request from the server. To mimic the RAD behaviour exactly in this situation I may have to force the language button to request a fresh copy every time its clicked. This will defeat the whole benefit of using react in the first place.  

I understand that it is a required functionality but i would suggest that it be made a subject of another ticket instead, probably a spike.

Reply:
However, i18n (the translation) is smart enough to recognise the default language in the browser and request for a page in that language. But i will like to investigate a work around in another ticket.

Reply:
It seems like there is need for further investigation on this. I'm not sure if it needs a Spike or if it can be done as ongoing work on this card. Whichever is more convenient I guess but would it be possible to default the language link to a standard URL specifying the locale and then overwrite this client side to resolve the link as it does here?